stage1/usr_from_coreos/coreos-common.mk

```diff
@@ -11,9 +11,13 @@ $(call setup-tmp-dir,CCN_TMPDIR)
 # systemd version in coreos image
 CCN_SYSTEMD_VERSION := v229
 # coreos image version
+ifeq ($(RKT_ACI_ARCH),aarch64)
+CCN_IMG_RELEASE := 1053.2.0
+else
 CCN_IMG_RELEASE := 1032.0.0
+endif
```

Comment:
@lucab Tested 1068.0.0 based arm64 stage1-coreos.aci and stage1-fly.aci and they seem to run OK.